packages/edge-gateway/test/content security policy.spec.js

```diff
@@ -0,0 +1,18 @@
+import { test, getMiniflare } from './utils/setup.js'
```

Comment:
I dunno man. I don't think us old developers are ready for filenames with spaces in them, even if the tooling is. 😊

Reply:
loool, I took some time to realize the filename had spaces instead of underscores. Not sure what I did there